File: 백준/Bronze/25893. Majestic 10/Majestic 10.js
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');
const n = Number(input[0]);
const result = [];
for(let i = 1; i <= n; i++){
    result.push(input[i]);
    const [a, b, c] = input[i].split(" ").map(Number).sort((a, b) => a - b);
    if(a >= 10) result.push("triple-double\n");
    else if(b >= 10) result.push("double-double\n");
    else if(c >= 10) result.push("double\n");
    else result.push("zilch\n");
}
console.log(result.join("\n").trim());

File: 백준/Bronze/25893. Majestic 10/Majestic 10.py
n = int(input())
for i in range(n) :
    stats = list(map(int, input().split()))
    print(format(" ".join(map(str, stats))))
    a, b, c = sorted(stats)
    if a >= 10 : print("triple-double\n")
    elif b >= 10 : print("double-double\n")
    elif c >= 10 : print("double\n")
    else : print("zilch\n")